starknet-artifacts/contracts/contract.cairo/contract_abi.json

```diff
@@ -0,0 +1,137 @@
+[
```

Comment:
Apart from adding the artifacts directory to `.gitignore`, you also ought to unindex the json files (https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign)